src/content/tools/devtools/legacy-inspector.md

```diff
@@ -0,0 +1,573 @@
+---
+title: Use the legacy Flutter inspector
+description: Learn how to use the legacy Flutter inspector to explore a Flutter app's widget tree.
+---
+
+<?code-excerpt path-base="visual_debugging/"?>
+
+:::note
```

Comment:
I'm wondering if this little note, at the top, is enough warning. Let's say that someone googles something and lands _inside_ this page. They wouldn't even see that note at the top. @parlough, is there a watermark or something we can do to make this more obvious? (I'd love a subtle-but-legible watermark.)